Installing fails because chart museum cannot be reached

*Error*
```
waiting for install to be ready, if this is the first time then it will take a while to download images
Error: Looks like "http://admin:admin@chartmuseum.default.192.168.64.7.nip.io" is not a valid chart repository or cannot be reached: Get http://admin:admin@chartmuseum.default.192.168.64.7.nip.io/index.yaml: dial tcp 192.168.64.7:80: getsockopt: connection refused
Error: Command failed  helm repo add releases http://admin:admin@chartmuseum.default.192.168.64.7.nip.io
```

Result of `jx version`:
```
pounder:~ jdumay$ jx version
WARNING: Failed to find draft version: exit status 1
WARNING: Client: &version.Version{SemVer:"v0.10.1", GitCommit:"486291360aeffa2fe0a131fbf0b4f3c601056b16", GitTreeState:"clean"}
2018/02/08 13:26:14 grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp: missing address"; Reconnecting to { <nil>}
2018/02/08 13:26:15 grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp: missing address"; Reconnecting to { <nil>}
2018/02/08 13:26:17 grpc: addrConn.resetTransport failed to create client transport: connection error: desc = "transport: dial tcp: missing address"; Reconnecting to { <nil>}
Error: cannot connect to draftd
NAME        VERSION                    
jx          1.0.53            
Jenkins X   0.0.55            
Kubernetes  v1.9.0            
Helm Client v2.8.0+g14af25f   
Helm Server v2.8.0+g14af25f   
Git         git version 2.16.1
```

*Full log*
```
pounder:~ jdumay$ jx uninstall; jx install
? Are you sure you wish to remove the Jenkins X platform from the 'default' namespace on cluster 'https://192.168.64.7:8443'? : Yes
release "jenkins-x" deleted
Jenkins X has been successfully uninstalled 
Lets set up a git username and API token to be able to perform CI / CD

? Do you wish to use i386 as the GitHub username for CI/CD pipelines: Yes


You can now configure a wildcard DNS pointing to the new loadbalancer address 192.168.64.7
? Domain 192.168.64.7.nip.io
Cloning the Jenkins X cloud environments repo to /Users/jdumay/.jx/cloud-environments
? A local Jenkins X cloud environments repository already exists, recreate with latest? Yes
Cloning the Jenkins X cloud environments repo to /Users/jdumay/.jx/cloud-environments
Counting objects: 239, done.
Compressing objects: 100% (25/25), done.
Total 239 (delta 12), reused 28 (delta 7), pack-reused 205
helm repo add jenkins-x http://chartmuseum.build.cd.jenkins-x.io
"jenkins-x" has been added to your repositories
rm -rf secrets.yaml.dec
helm repo add jenkins-x http://chartmuseum.build.cd.jenkins-x.io
"jenkins-x" has been added to your repositories
helm repo update
Hang tight while we grab the latest from your chart repositories...
...Skip local chart repository
...Successfully got an update from the "jenkins-x" chart repository
...Successfully got an update from the "incubator" chart repository
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈ Happy Helming!⎈ 
helm install jenkins-x/jenkins-x-platform --name jenkins-x -f ./myvalues.yaml -f ./secrets.yaml --version 0.0.55 --values=/Users/jdumay/.jx/gitSecrets.yaml --values=/Users/jdumay/.jx/extraValues.yaml --namespace=default
NAME:   jenkins-x
LAST DEPLOYED: Thu Feb  8 13:03:13 2018
NAMESPACE: default
STATUS: DEPLOYED

RESOURCES:
==> v1/ServiceAccount
NAME                SECRETS  AGE
cleanup             1        32s
expose              1        32s
pipelinecontroller  1        32s

==> v1/Service
NAME                            TYPE       CLUSTER-IP      EXTERNAL-IP  PORT(S)    AGE
jenkins-x-chartmuseum           ClusterIP  10.102.116.155  <none>       8080/TCP   32s
jenkins-x-docker-registry       ClusterIP  10.105.56.116   <none>       5000/TCP   32s
heapster                        ClusterIP  10.99.17.32     <none>       8082/TCP   32s
jenkins-agent                   ClusterIP  10.108.54.139   <none>       50000/TCP  32s
jenkins                         ClusterIP  10.103.220.210  <none>       8080/TCP   32s
jenkins-x-kubernetes-dashboard  ClusterIP  10.98.8.147     <none>       80/TCP     32s
jenkins-x-mongodb               ClusterIP  10.96.97.148    <none>       27017/TCP  32s
jenkins-x-monocular-api         ClusterIP  10.107.106.179  <none>       80/TCP     32s
jenkins-x-monocular-prerender   ClusterIP  10.103.210.5    <none>       80/TCP     32s
jenkins-x-monocular-ui          ClusterIP  10.106.230.62   <none>       80/TCP     32s
jenkins-x-sonatype-nexus        ClusterIP  10.105.254.76   <none>       8081/TCP   32s

==> v1beta1/Deployment
NAME                            DESIRED  CURRENT  UP-TO-DATE  AVAILABLE  AGE
jenkins-x-chartmuseum           1        1        1           1          32s
jenkins-x-docker-registry       1        1        1           1          32s
jenkins-x-heapster              1        1        1           1          32s
jenkins                         1        1        1           0          32s
jenkins-x-kubernetes-dashboard  1        1        1           1          32s
jenkins-x-mongodb               1        1        1           1          32s
jenkins-x-monocular-api         1        1        1           0          32s
jenkins-x-monocular-prerender   1        1        1           1          32s
jenkins-x-monocular-ui          1        1        1           0          32s
pipelinecontroller              1        1        1           1          32s
jenkins-x-sonatype-nexus        1        1        1           0          32s

==> v1beta1/CustomResourceDefinition
NAME                  AGE
pipelines.jenkins.io  32s
runs.jenkins.io       32s

==> v1/Pod(related)
NAME                                             READY  STATUS   RESTARTS  AGE
jenkins-x-chartmuseum-d48f95c7d-ttfn9            1/1    Running  0         32s
jenkins-x-docker-registry-794dcb698-h2tcv        1/1    Running  0         32s
jenkins-x-heapster-76df87bd59-47g4j              2/2    Running  0         28s
jenkins-77f44b4c76-hzc96                         0/1    Running  0         32s
jenkins-x-kubernetes-dashboard-6b87d747b7-p8krt  1/1    Running  0         32s
jenkins-x-mongodb-6c84f866dc-j24j8               1/1    Running  0         32s
jenkins-x-monocular-api-6548574cb5-xvb8d         0/1    Running  1         32s
jenkins-x-monocular-prerender-6d8897856-ghzsf    1/1    Running  0         32s
jenkins-x-monocular-ui-7dccdf6c9b-g2fls          0/1    Running  0         32s
pipelinecontroller-5dbc96bbd4-ft5zp              1/1    Running  0         31s
jenkins-x-sonatype-nexus-5b976bd546-cdk2d        0/1    Running  0         31s

==> v1/Secret
NAME                              TYPE    DATA  AGE
jenkins-x-docker-registry-secret  Opaque  1     33s
jenkins                           Opaque  2     33s
jenkins-x-mongodb                 Opaque  2     33s
jenkins-docker-cfg                Opaque  1     33s
jenkins-git-credentials           Opaque  1     33s
jenkins-hub-api-token             Opaque  1     33s
jenkins-git-ssh                   Opaque  2     33s
jx-basic-auth                     Opaque  1     33s
jenkins-release-gpg               Opaque  4     33s
jenkins-maven-settings            Opaque  1     33s
jenkins-npm-token                 Opaque  1     33s
jenkins-ssh-config                Opaque  1     33s

==> v1/ConfigMap
NAME                              DATA  AGE
jenkins-x-docker-registry-config  1     33s
jenkins                           3     33s
jenkins-tests                     1     33s
jenkins-x-monocular-api-config    1     33s
jenkins-x-monocular-ui-config     1     33s
jenkins-x-monocular-ui-vhost      1     33s

==> v1/PersistentVolumeClaim
NAME                      STATUS  VOLUME                                    CAPACITY  ACCESS MODES  STORAGECLASS  AGE
jenkins-x-chartmuseum     Bound   pvc-3893faeb-0c74-11e8-8c25-26c3e276d95b  8Gi       RWO           standard      33s
jenkins                   Bound   pvc-3894f09c-0c74-11e8-8c25-26c3e276d95b  3Gi       RWO           standard      33s
jenkins-x-mongodb         Bound   pvc-3896f70c-0c74-11e8-8c25-26c3e276d95b  8Gi       RWO           standard      33s
jenkins-x-sonatype-nexus  Bound   pvc-3897bfec-0c74-11e8-8c25-26c3e276d95b  4Gi       RWO           standard      32s


waiting for install to be ready, if this is the first time then it will take a while to download images
Error: Looks like "http://admin:admin@chartmuseum.default.192.168.64.7.nip.io" is not a valid chart repository or cannot be reached: Get http://admin:admin@chartmuseum.default.192.168.64.7.nip.io/index.yaml: dial tcp 192.168.64.7:80: getsockopt: connection refused
Error: Command failed  helm repo add releases http://admin:admin@chartmuseum.default.192.168.64.7.nip.io
```

I see this: 

```
Update Complete. ⎈ Happy Helming!⎈ 
helm install jenkins-x/jenkins-x-platform --name jenkins-x -f ./myvalues.yaml -f ./secrets.yaml --version 0.0.55 --values=/Users/michaelneale/.jx/gitSecrets.yaml --values=/Users/michaelneale/.jx/extraValues.yaml --namespace=default
Error: timed out waiting for the condition
make: *** [install] Error 1
Error: Command failed  make ARGS=--values=/Users/michaelneale/.jx/gitSecrets.yaml --values=/Users/michaelneale/.jx/extraValues.yaml --namespace=default install
error: exit status 2
```

on jx install (jx install can't be tried again, as "a release named jenkins-x already exists."

--- a/README.md
+++ b/README.md
@@ -42,6 +42,11 @@ To start Kubernetes:
 
     minikube start --vm-driver hyperkit --memory 4028
     
+You will also need to enable the ingress addon
+
+    minikube addons enable ingress
+    
+
 ### Setup Jenkins X
 
 Then ensure that Helm is initialized on your Kubernetes cluster by running:

--- a/README.md
+++ b/README.md
@@ -42,7 +42,7 @@ To start Kubernetes:
 
     minikube start --vm-driver hyperkit --memory 4028
     
-You will also need to enable the ingress addon
+Once Kubernetes is started, yoiu need to enable the ingress addon (run once after installation):
 
     minikube addons enable ingress
     

